src/content/datachannel/messaging/main.js

```diff
@@ -0,0 +1,164 @@
+/*
+ *  Copyright (c) 2018 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+/* eslint no-unused-expressions: 0 */
+
+'use strict';
+import {LitElement, html} from 'https://unpkg.com/@polymer/lit-element?module';
+
+class MessagingSample extends LitElement {
+  constructor() {
+    super();
+    this._localConnection = this._remoteConnection = this._localChannel = this._remoteChannel = null;
```

Comment:
If you never check if these are null (they are undefined by default) why do this?

Reply:
Anyway they look nice on each on a separate line